features/environment.py

```diff
@@ -28,6 +28,7 @@
     'immunizations',
     'patient-documents',
 }
+DEFAULT_ES_URL = 'https://search-s4s-logs-xsjsafiwd7vkpiucmjqmdjkp7y.us-west-2.es.amazonaws.com/test-suite/log/'
```

Comment:
Same here, I think -- default should be no logging (especially since the test suite could be used in-house by vendors)